### PR TITLE
Fixes ANSI character output regression on Windows

### PR DESCRIPTION
**Pull request by LorenzCK**
Fixes a regression introduced in Hugo version 0.50 by commit df021317a964a482cd1cd579de5a12d50faf0d08, related to issue #4462.

Previously this was fixed in 16b1a4148ad05a40cd748e373b5dc18447285837 by replacing the ANSI characters in `commands/helpers.go` with empty strings. This fix instead updates package `common/terminal`, thus disabling all ANSI character output on Windows (including output colorization).

### CODE REVIEW

File: common/terminal/colors_others.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"fmt"
+	"os"
+
+	isatty "github.com/mattn/go-isatty"
+)
+
+// IsTerminal return true if the file descriptor is terminal and the TERM
+// environment variable isn't a dumb one.
+func IsTerminal(f *os.File) bool {
+	fd := f.Fd()
```
Comment (bep) [inline common/terminal/colors_others.go]: Just return false if `runtime.GOOS == "windows", which is a smaller and better change.

Comment (LorenzCK) [reply common/terminal/colors_others.go]: OK, changes applied. Please squash if this is OK 😊

File: common/terminal/colors.go [hunk 2]
```diff
@@ -52,6 +57,10 @@ func Warning(s string) string {
 
 // colorize s in color.
 func colorize(s, color string) string {
+	if runtime.GOOS == "windows" {
```
Comment (bep) [inline common/terminal/colors.go]: This check is superflous. The `IsTerminal` is always called first. We will eventually try to get "some Windows support", so having the check in one place only would make that easier.

Comment (LorenzCK) [reply common/terminal/colors.go]: OK, done.

**Comment by bep** [comment]
Merged and thanks! For future PRs, see my slightly modified commit message (we have some guidelines somewhere).

**Comment by LorenzCK** [comment]
Great, thanks @bep, will check out the guidelines

**Pull request merged**